ci/deployment.yaml

```diff
@@ -102,6 +102,9 @@ spec:
            - mountPath: /sql-config
              name: sql-config
              readOnly: true
+           - mountPath: /secrets/oauth-token
+             name: hail-ci-0-1-github-oauth-token
```

Comment:
What do we store here though? Do we have a corresponding "robot" account? Which permissions does it need?

Reply:
Following DanG's email 9 days ago, we could set up a ci_config section in our terraform vars:

> I wanted- To enable CI, there is now a new top-level terraform variable called `ci_config`. Including this in your `global.tfvars` will create the necessary resources to run CI. I would like at the variable in /infra/gcp/main.tf to gain an understanding of how to use the configuration. Here’s an example of what mine looked like:

```
ci_config = {
  github_oauth_token = "XXXX"
  github_user1_oauth_token = "XXXX"
  watched_branches = [["daniel-goldstein/hail:main", true, false]]  # This `false` here will prevent the CI from merging PRs into the repo
  deploy_steps = ["deploy_batch", "test_batch_0", "deploy_ci"]
  bucket_location = "US"
  bucket_storage_class = "MULTI_REGIONAL"
  github_context = "ci-gcp"
}```

It should probably be separate from our `infra/global.tfvars` config that we maintain under git, e.g.`~/.hail/ci.tfvars`. I set up one using my own GitHub user just to see how it works. But we can switch to a robot account of course!